README.md

```diff
@@ -652,3 +652,9 @@ See [Settings](#settings) above for examples and to understand what you can conf
 The server and client are the user-facing components of the API system, but there are a number of other components that work together to make sure your settings are applied, and that they survive upgrades of Bottlerocket.
 
 For more details, see the [API system documentation](sources/api/).
+
+### Default Volumes
+
+Bottlerocket operates with two default storage volumes. 
+* The root device, `/dev/xvda`, is immutable and is mostly used for holding the OS images with its two partitions. It also contains the bootloader and the dm-verity hash tree for verifying the root filesystem image.
```

Comment:
nits:
* one sentence per line
* there are two partition sets that each consist of multiple partitions, rather than two partitions
* settings store is also on the root device

```suggestion
* The root device, `/dev/xvda`, is immutable and is mostly used for holding the OS images with its active and passive partition sets.
  It also contains the bootloader, the dm-verity hash tree for verifying the root filesystem image, and the data store for the Bottlerocket API.
```